packages/web-components/fast-foundation/src/calendar/calendar.template.ts

```diff
@@ -79,6 +79,7 @@ export function calendarCellTemplate(
                     `${x.month}-${x.day}-${x.year}`,
                     { month: "long", day: "numeric" }
                 )}"
+            aria-selected="${(x, c) => x.selected}"
```

Comment:
Does the cell receive focus by keyboard when interactive? If not, I'm not sure I'd expect this here. https://www.w3.org/WAI/ARIA/apg/patterns/grid/#gridNav_focus

Reply:
The cell receives focus by keyboard so the user can navigate on the calendar and select a date. Here's the React v8 Calendar for reference: https://developer.microsoft.com/en-us/fluentui#/controls/web/calendar

Not sure if this is what you're asking about though. I looked at the link you sent me, and now I'm wondering if the div inside the cell should have a button role instead, since it's interactive. Otherwise, I think the calendar would fall under "A cell contains text or a single graphic and grid navigation keys set focus on the cell."

What do you think should be the proper role here?